exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceRuntime.java

```diff
@@ -386,6 +389,37 @@ public void connectServiceApis(int[] serviceIds, Node node) {
     }
   }
 
+  /**
+   * Get service instance by its name.
+   *
+   * @throws IllegalArgumentException if there is no service with such name
+   */
+  public Service getServiceInstanceByName(String serviceName) {
+    return findService(serviceName)
```

Comment:
This class is specified as thread-safe. The new public methods must keep that promise.